### PR TITLE
Migrate to Scalafmt 0.4.7

### PR DESCRIPTION
**Pull request by densh**

### CODE REVIEW

**Comment by densh** [comment]
@olafurpg `sbt scalafmtTest` after `sbt scalafmt` fails. Any ideas?

**Comment by olafurpg** [comment]
Hmmmm. Did you clean before running `scalafmt`?

File: scripted-tests/run/string-from-to-ctring/project/scala-native.sbt [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   val pluginVersion = System.getProperty("plugin.version")
-  if(pluginVersion == null)
-    throw new RuntimeException("""|The system property 'plugin.version' is not defined.
+  if (pluginVersion == null)
+    throw new RuntimeException(
+      """|The system property 'plugin.version' is not defined.
```
Comment (olafurpg) [inline scripted-tests/run/string-from-to-ctring/project/scala-native.sbt]: This is a bit of a shame, `assumeStandardLibraryStripMargin = true` would fix it.

Comment (densh) [inline scripted-tests/run/string-from-to-ctring/project/scala-native.sbt]: Added to the config.

**Comment by olafurpg** [comment]
Yay 🎉

**Pull request merged**